test/dynamicppl/model.jl

```diff
@@ -0,0 +1,63 @@
+module DynamicPPLModelTests
+
+using Test: @testset, @test
+using Turing
+
+# TODO(penelopeysm): Move this to DynamicPPL Test Utils module
```

Comment:
This method depends on `MCMCChains.jl` though, so need to find a way around it.

Reply:
Looking at this file more carefully, shouldn't we be able to move all of this to DynamicPPLMCMCChainsExt too?

Reply:
Moved all of these to DynamicPPL tests 😄